jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_sorting_view.py

```diff
@@ -93,8 +93,8 @@ def _handle_message(self, widget, msg, buffers):
             elif msg['type'] == 'addSubfeedWatch':
                 self._worker_session.add_subfeed_watch(
                     watch_name=msg['watchName'],
-                    feed_id=msg['feedId'],
-                    subfeed_hash=msg['subfeedHash']
+                    feed_uri=msg['feedUri'],
```

Comment:
Pass feed_uri and subfeed_name rather than feed_id and subfeed_hash.

Note that feed_uri and subfeed_hash can always be derived from feed_id and subfeed_name.